lib/utils/dataflow/src/index.js

```diff
@@ -319,12 +327,27 @@ const postOutput = function *(odoc,
   }
 };
 
+// Check if any of the docs have errors in it
+const validateDocs = (docs) => find(docs, (doc) => doc.error) ? [] : docs;
+
 // Post a list of output docs to the configured sink service
 const postOutputs = function *(odocs,
   shost, spartition, spost, authentication) {
-  yield tmap(odocs, function *(odoc, i, l) {
```

Comment:
Why do we need to change a map to a reduce here? Looks like this will have a significant impact on performance, as we won't be able to parallelize the posts anymore...